aws/ec2.go

```diff
@@ -27,7 +29,9 @@ func filterOutProtectedInstances(svc *ec2.EC2, output *ec2.DescribeInstancesOutp
 			protected := *attr.DisableApiTermination.Value
 			// Exclude protected EC2 instances
 			if !protected {
-				filteredIds = append(filteredIds, &instanceID)
+				if excludeAfter.After(*instance.LaunchTime) {
```

Comment:
I believe this code originally had a bug in it that Josh caught. Is it possible to add a test case that would've caught that?

Reply:
Tests added